### PR TITLE
amazon-ebs update simple example

### PR DESCRIPTION
**Pull request by kikitux**
update exaple for amazon-ebs builder

make simple example to be complete to make it easier for newcomers

### CODE REVIEW

**Comment by rickard-von-essen** [comment]
This is equivalent to:

```json
    {
        "type": "amazon-ebs",
        "region": "us-east-1",
        "source_ami": "ami-fce3c696",
        "instance_type": "t2.micro",
        "ssh_username": "ubuntu",
        "ami_name": "packer-quick-start {{timestamp}}"
    }
```

IMHO it's better to drop the altogether.

File: website/source/docs/builders/amazon-ebs.html.md [hunk 1]
```diff
@@ -549,14 +549,20 @@ run:
...
-  "region": "us-east-1",
-  "source_ami": "ami-fce3c696",
-  "instance_type": "t2.micro",
-  "ssh_username": "ubuntu",
-  "ami_name": "packer-quick-start {{timestamp}}"
+    "variables": {
+        "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+        "aws_secret_access_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
+    },
+    {
```
Comment (azr) [inline website/source/docs/builders/amazon-ebs.html.md]: Ah, this probably is a builder entry:
```suggestion
    "builders": [
        {
```

File: website/source/docs/builders/amazon-ebs.html.md [hunk 2]
```diff
@@ -549,14 +549,20 @@ run:
...
+        "type": "amazon-ebs",
+        "access_key": "{{user `aws_access_key`}}",
+        "secret_key": "{{user `aws_secret_access_key`}}",
+        "region": "us-east-1",
+        "source_ami": "ami-fce3c696",
+        "instance_type": "t2.micro",
+        "ssh_username": "ubuntu",
+        "ami_name": "packer-quick-start {{timestamp}}"
+    }
```
Comment (azr) [inline website/source/docs/builders/amazon-ebs.html.md]: I hope this is correct:
```suggestion
    ]}
```

**Comment by kikitux** [comment]
@rickard-von-essen i am making the example ready to copy/paste for new users.

**Comment by kikitux** [comment]
@azr sorry! i fixed the sample, i should have check with `jq`

```json
{
  "variables": {
    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}"
  },
  "builders": [
    {
      "type": "amazon-ebs",
      "access_key": "{{user `aws_access_key`}}",
      "secret_key": "{{user `aws_secret_key`}}",
      "region": "us-east-1",
      "source_ami": "ami-fce3c696",
      "instance_type": "t2.micro",
      "ssh_username": "ubuntu",
      "ami_name": "packer_AWS {{timestamp}}"
    }
  ]
}

```

**Comment by azr** [comment]
No problemo @kikitux 🙂 

@rickard-von-essen: Ah, we commented at the same time; I think it's good to be more explicit and to show other features of packer in docs.
Users that know nothing will probably just copy paste and then understand.
More experienced users will probably just copy paste what they need.

**Comment by SwampDragons** [comment]
yeah, I think it's better to have it explicitly clear in the example where the credentials are coming from, even if experienced users know they can leave the variables section out in this case.

**Pull request merged**